openadapt/strategies/mixins/svg.py

```diff
@@ -0,0 +1,41 @@
+"""
+Implements a ReplayStrategy mixin using VTracer to convert an image to SVG text.
+
+Usage:
+
+    class MyReplayStrategy(SVGReplayStrategyMixin):
+        ...
+"""
+import subprocess
+import tempfile
+
+from openadapt.models import Recording, Screenshot
+from openadapt.strategies.base import BaseReplayStrategy
+
+
+class SVGReplayStrategyMixin(BaseReplayStrategy):
+    """ReplayStrategy mixin that uses VTracer to convert an image to SVG text.
+
+    Attributes:
+        recording: the recording to be played back
+
+    """
+    def __init__(self, recording: Recording):
+        super().__init__(recording)
```

Comment:
Before calling `super()`, we can check to see if rust and vtracer are installed, and if not then install them.

Reply:
addressed in [3eb0496](https://github.com/OpenAdaptAI/OpenAdapt/pull/198/commits/3eb049623e7ce090b290652c231f42146150c1e5)